content/components/banner.mdx

```diff
@@ -8,15 +8,256 @@ tags:
   - flash
 ---
 
+import {Box} from '@primer/react'
 import ComponentLayout from '~/src/layouts/component-layout'
 export default ComponentLayout
 import {AccessibilityLink} from '~/src/components/accessibility-link'
 
-Banner can be used in one of two ways:
+<img
+  width="960"
+  alt="Image displaying each variant of banenr, from top: critical, warning, success, info, upsell."
+  src="https://github.com/primer/primitives/assets/813754/f96b1eda-2413-4fc1-b1a7-3775b0c3b324"
+/>
+
+## Usage
 
+Banner can be used in one of two ways:
 1. To highlight prominent information on a page.
 2. To communicate feedback in response to a user action.
 
+For system wide information, consider using an [announcement](../ui-patterns/messaging#message-types).
+
+## Anatomy
+
+<img
+  width="960"
+  alt="Anatomy of a banner, including the message, title, icon, dismiss button, primary, and secondary action"
+  src="https://github.com/primer/design/assets/813754/defa52f8-49bb-4332-aaa1-872a22a563a9"
```

Comment:
I'd recommend changing this diagram to only contain a single banner with all parts labeled.

The purple banner in this image has all the parts we'd need.

Reply:
Agree with Mike 👍